DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/src/mvdr_beamforming.cpp

```diff
@@ -245,6 +245,9 @@ int main(int argc, char *argv[]) {
     // read the input data
     passed &=
         ReadInputData(in_dir, (ComplexType *)in_data.data(), num_matrix_copies);
+    if (!passed) {
+      std::terminate();
```

Comment:
Maybe print an error here that we failed to read? Or is that printed in ReadInputData()?

Reply:
It is printed in ReadInputData (glad I'm not the only lazy reviewer :-) )